### ISSUE TITLE
Softmax in inference.py

### ISSUE THREAD

**Issue by ilteralp** [yassouali/pytorch_segmentation#102]
Hi all! Thank you for the good work. I have a question, why a `softmax` is used in `inference.py` ([this line](https://github.com/yassouali/pytorch_segmentation/blob/master/inference.py#L155)) ? It was not explicitly used in `train.py`.

**Comment by luczeng** [comment]
I think it's because the trainer uses `CrossEntropy` from torch which has the softmax embedded by default.
